index.js

```diff
@@ -25,15 +27,40 @@ class TaskList {
       description.classList.remove("input-error");
     }
     if (!title.value || !description.value) return;
-    this.tasks.push(new Task(title.value, description.value));
+    this.tasks.push(new Task(title.value, description.value, priority.value));
     title.value = "";
     description.value = "";
+    priority.value = "low";
+    this.changeTaskPriorityIndicator();
     console.log(this.tasks);
   }
+  changeTaskPriorityIndicator() {
+    const selectedPriority = document.getElementById("task-priority").value;
+    const background = document.getElementById("task-priority-panel");
+    switch (selectedPriority) {
+      case "low":
+        background.style.backgroundColor = "lightgreen";
+        break;
+      case "medium":
+        background.style.backgroundColor = "yellow";
+        break;
+      case "high":
+        background.style.backgroundColor = "darkred";
+        break;
+      default:
+        background.style.backgroundColor = "white";
+        break;
+    }
+  }
 }
 
 document.addEventListener("DOMContentLoaded", function () {
   const taskList = new TaskList();
   const btn = document.getElementById("add-task-btn");
   btn.addEventListener("click", taskList.addTask.bind(taskList));
+  const prioritySelect = document.getElementById("task-priority");
+  prioritySelect.addEventListener(
+    "change",
+    taskList.changeTaskPriorityIndicator.bind(taskList)
```

Comment:
Why are you using `bind`  when calling the class methods?

Can you get back to the callback structure and remember how we used it? [`it's a function`]